c++/src/kj/time.c++

```diff
@@ -268,6 +268,9 @@ CappedArray<char, sizeof(int64_t) * 3 + 2 + 4> KJ_STRINGIFY(Date d) {
   return kj::toCharSequence(d - UNIX_EPOCH);
 }
 CappedArray<char, sizeof(int64_t) * 3 + 2 + 4> KJ_STRINGIFY(Duration d) {
```

Comment:
Not important, but I'm not actually sure exactly what the `n*3 + 2 + 4` is intended to contain... I guess `n*3` is meant to cover the digits themselves, and the `2` is maybe to cover the two-character suffix.  And the `4` to cover the decimal point, null character, and maybe some padding for safety?  Not sure if the upper-ascii micro symbol incurs extra space in some environments or something.  A moot point here, though, since there seems to be plenty of room in the digits estimation to accommodate an additional negative sign at maxint.

Reply:
Oh yeah, the μ takes two bytes (UTF-8). I probably didn't think of that when writing this, but it does appear I added extra padding anyway. I'm going to replace `2 + 4` with `8` to be safe (and less confusing).